test/CMakeLists.txt

```diff
@@ -232,7 +240,12 @@ if (BENCHMARK_ENABLE_GTEST_TESTS)
 
   macro(add_gtest name)
     compile_gtest(${name})
-    add_test(NAME ${name} COMMAND ${name})
+    benchmark_add_test(NAME ${name} COMMAND ${name})
+    if(WIN32 AND BUILD_SHARED_LIBS)
+      set_tests_properties(${name} PROPERTIES
```

Comment:
this feels like something CMake should be taking care of with the dependencies, no?

Reply:
I don't know, this is just how things work on windows i guess.
I agree that it's very broken, and i don't use windows myself too.